recipes/strling/0.1.0/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
Can you try to compile the tool at install? This is generally preferred in Bioconda over packaging a pre-compiled binary. That would also allow you to get a macOS version (the macOS build is currently failing because the binary is for Linux only).

Reply:
I tried hard. There suggested way of using nimble does not work because of type int64 vs. int mismatches. I could no figure out how to solve this problem.

Reply:
Is the binary statically linked? Then I think it’s ok for now.

Reply:
And add 'skip: True  # [osx]` so the macOS package isn’t built.